CMakeLists.txt

```diff
@@ -70,7 +70,8 @@ set(PCG_DIRECTORY ${Galois_SOURCE_DIR}/external/pcg-cpp/include)
 
 add_subdirectory(microbench)
 add_subdirectory(scripts)
-add_subdirectory(wf4)
```

Comment:
was wf4 broken by the changes or was this an accident?

Reply:
@divija95 did you remove wf4 on purpose?

Reply:
Yes, wf4 has some incompatible apis wrt DistLocalGraph
ToDO: @divija95 to fix it